draft-ietf-oauth-jwk-thumbprint-uri.xml

```diff
@@ -102,10 +102,21 @@
 	</list>
       </t>
       <t>
-	The prefix is followed by a colon and a JWK Thumbprint value to form
-	a URI representing a JWK Thumbprint.
+	The prefix is followed by a hash algorithm identifier and a JWK Thumbprint value, 
+  separated by the colon to form a URI representing a JWK Thumbprint.
       </t>
 
+      <section anchor="HashAlgorithms" title="Hash Algorithms Identifier">
+  <t>
+  Hash algorithm identifier used in JWK Thumbprint URIs are strings 
+  registered in the IANA "Named Information Hash Algorithm Registry" <xref target="IANA.Hash.Algorithms"/>.
+  </t>
+
+  <section anchor="MTI" title="Mandatory to Implement Hash Algorithm">
+	<t>
+	  Mandatory to implement Hash Algorithm is SHA-256.
```

Comment:
How about this?  "To promote interoperability among implementations, the SHA-256 hash algorithm is mandatory to implement."  That way we're saying why there's an MTI algorithm.